client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/package-row.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { map, uniq } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ItemRow, { Header as ItemRowHeader } from './item-row';
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import Dropdown from 'woocommerce/woocommerce-services/components/dropdown';
+import Checkbox from 'woocommerce/woocommerce-services/components/checkbox';
+import {
+	setContentsType,
+	setContentsExplanation,
+	setRestrictionType,
+	setRestrictionExplanation,
+	setAbandonOnNonDelivery,
+	setITN,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+const PackageRow = ( props ) => {
+	const {
+		siteId,
+		orderId,
+		errors,
+		packageId,
+		translate,
+		contentsType,
+		contentsExplanation,
+		restrictionType,
+		restrictionExplanation,
+		abandonOnNonDelivery,
+		itn,
+		items,
+	} = props;
+	const abandonHandler = () => props.setAbandonOnNonDelivery( ! abandonOnNonDelivery );
+
+	return <div className="customs-step__package">
+		<label htmlFor={ packageId + '_abandonOnNonDelivery' }>
+			<Checkbox
+				id={ packageId + '_abandonOnNonDelivery' }
+				checked={ ! abandonOnNonDelivery }
+				onChange={ abandonHandler } />
+			<span>{ translate( 'Return to sender if package is unable to be delivered' ) }</span>
+		</label>
+
+		<div className="customs-step__restrictions-row">
+			<div className="customs-step__contents-type">
+				<Dropdown
```

Comment:
Partially related: I noticed that the dropdown label (e.g. 'Contents type') wasn't being read out by my screen reader (e.g. ChromeVox). This led me to find out that we're incorrectly using a `<FormLegend />` instead of `<FormLabel />` to label the dropdown form elements: https://github.com/Automattic/wp-calypso/blob/03ddf9aaa09ae9af56338da57c54e5a4cef1b5eb/client/extensions/woocommerce/woocommerce-services/components/dropdown/index.js#L33

Labels are used for labelling one element, whereas legends are used for labelling a collection of form elements ([MDN](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/How_to_structure_an_HTML_form)).

Not a blocker for the release, so we should either fix or make an issue.

Reply:
I've created https://github.com/Automattic/wp-calypso/issues/26302 to track this. We are using `<FormLegend>` in deveral other places (not only dropdowns) and we should fix them all at once.